ext/civiimport/templates/CRM/Import/Monitor.tpl

```diff
@@ -0,0 +1,17 @@
+<div id='crm-import-progress'></div>
+{literal}
+  <script>
+    CRM.$(function($) {
+      var target = '#crm-import-progress';
+      var url = CRM.vars.civiimport.url;
+      // Load the snippet into the container
+      CRM.loadPage(url, {
+        target: target,
+        block: false
+      })
+      window.setInterval(function () {
+        CRM.$(target).crmSnippet('refresh');
```

Comment:
This is a lot of hits on the server. Best skip if the user isn't paying attention...

```suggestion
        if (document.hasFocus()) {
          CRM.$(target).crmSnippet('refresh');
        }
```

Reply:
ohh - good idea

Reply:
Done